packages/discord.js/src/structures/Role.js

```diff
@@ -193,6 +193,10 @@ class Role extends Base {
    * @param {RoleResolvable} role Role to compare to this one
    * @returns {number} Negative number if this role's position is lower (other role's is higher),
    * positive number if this one is higher (other's is lower), 0 if equal
+   * * @example
+   * // Compare the position of a role to another
+   * const comparisation = comparedRole.comparePositionTo(roleToCompareTo)
+   * if (comparisation === 1) console.log(`${comparedRole.name} is higher than ${roleToCompareTo.name}`)
```

Comment:
```suggestion
   * if (comparisation > 1) console.log(`${comparedRole.name} is higher than ${roleToCompareTo.name}`)
```

Also it could check every statement ( higher, lower, equal )